usaspending_api/api_contracts/contracts/v2/idvs/awards.md

```diff
@@ -117,10 +117,10 @@ List child IDVs, child awards, or grandchild awards for IDV
     Unique internal surrogate identifier for an award.  Deprecated.  Use `generated_unique_award_id`.
 + `award_type`: `BPA`(required, string)
 + `description`: `OTHER THAN SCHEDULE,IGF::OT::IGF` (required, string, nullable)
-+ `funding_agency`: `GENERAL SERVICES ADMINISTRATION (GSA)` (required, string)
-+ `awarding_agency`: `GENERAL SERVICES ADMINISTRATION (GSA)` (required, string)
-+ `funding_agency_id`: 634 (required, number)
-+ `awarding_agency_id`: 634 (required, number)
++ `funding_agency`: `GENERAL SERVICES ADMINISTRATION (GSA)` (required, string, nullable)
++ `awarding_agency`: `GENERAL SERVICES ADMINISTRATION (GSA)` (required, string, nullable)
++ `funding_agency_id`: 634 (required, number, nullable)
++ `awarding_agency_id`: 634 (required, number, nullable)
```

Comment:
Can all four of these be null?

I thought the awarding agency was a required field for transactions

Reply:
Whether or not they should be, some are null.  And some transactions have invalid subtier codes so we couldn't link them to an agency even if we wanted to.

Reply:
That makes sense, but we might need to confirm it is desired behavior.